package.json

```diff
@@ -56,16 +56,17 @@
     "webpack-dev-server": "^3.8.0"
   },
   "dependencies": {
+    "@openmrs/esm-api": "^1.0.3",
     "@openmrs/react-root-decorator": "^1.0.0",
     "@reach/tabs": "^0.1.6",
     "@types/jest": "^24.0.18",
     "@types/react": "^16.9.2",
     "@types/react-dom": "^16.9.0",
     "@types/single-spa-react": "^2.8.3",
-    "@types/systemjs": "^0.20.6",
     "kremling": "^2.0.1",
     "react": "^16.9.0",
     "react-dom": "^16.9.0",
+    "semver": "^6.3.0",
```

Comment:
`semver` should be a `devDependency`.

`dependencies` are for "in-browser" dependencies. `devDependencies` are for everything else.